modules/dnn/include/opencv2/dnn/layer.hpp

```diff
@@ -75,6 +78,7 @@ class CV_EXPORTS LayerFactory
 
 private:
     LayerFactory();
+    static std::map<String, int>& _getRegisteredLayers();
```

Comment:
Remove that from public headers.
All "private" members should be eliminated from public header, see https://github.com/opencv/opencv/wiki/Coding_Style_Guide

Reply:
Corrected, added [layer_reg.private.hpp](https://github.com/opencv/opencv/pull/19693/files#diff-72def3133939a67ff89fd1fe6b97e94f4bb697472f71aa7de45d822998b4499c).